pkg/reconciler/taskrun/validate_resources.go

```diff
@@ -120,7 +120,7 @@ func ValidateResolvedTaskResources(params []v1alpha1.Param, rtr *resources.Resol
 		outputs = rtr.TaskSpec.Resources.Outputs
 	}
 	if err := validateResources(inputs, rtr.Inputs); err != nil {
-		return fmt.Errorf("invalid input resources: %w", err)
+		return fmt.Errorf("invalid input resources for task %s: %w", rtr.TaskName, err)
 	}
 	if err := validateResources(outputs, rtr.Outputs); err != nil {
 		return fmt.Errorf("invalid output resources: %w", err)
```

Comment:
We should probably do it for output resources too :wink:

Reply:
And for `params` too 😉 

```
return fmt.Errorf("invalid input params: %w", err)
```

to

```
return fmt.Errorf("invalid input params for task %s: %w", rtr.TaskName, err)
```

Reply:
thanks @jlpettersson 👍